### DISCUSSION TITLE
Fehlermelung

### DISCUSSION THREAD

**Discussion by MatthiasGruening** [kimai/kimai#3926] [On-Premise: Installation & Updates]
Hallo, ich hoffe, es kann mir jemand helfen. Ich habe Kimai installiert, was alles funktioniert hat. Nun versuche ich Plugins zu installieren und folgender Fehler wird erzeugt:

In DefinitionErrorExceptionPass.php line 54:
                                                                               
  Cannot autowire service "KimaiPlugin\VacationBundle\Controller\PublicHolida  
  yController": argument "$metaFieldRuleRepository" of method "__construct()"  
   has type "KimaiPlugin\MetaFieldsBundle\Repository\MetaFieldRuleRepository"  
   but this class was not found.                                               
                                                                               

Was mache ich falasch? 
Viele Grüße
Matthias

**Comment by kevinpapst** [comment]
This error tells you, that the MetaFieldBundle (Custom fields plugin) is missing.

Vacation bundle is a paid plugin by a third party developer, whome you should call out for help next time.

**Reply by MatthiasGruening** [reply]
I solved the problem, thanks.
